platine-management-api/src/main/java/fr/insee/survey/datacollectionmanagement/contact/controller/ContactController.java

```diff
@@ -135,6 +135,15 @@ public ResponseEntity<ContactDto> putContact(@PathVariable("id") String id,
 
     }
 
+    @Operation(summary = "Give questioning accreditation to contact as main")
+    @PutMapping(value = UrlConstants.API_MAIN_CONTACT_INTERROGATIONS_ASSIGN)
+    @PreAuthorize(AuthorityPrivileges.HAS_MANAGEMENT_PRIVILEGES)
+    public void updateInterrogationToMainContactAsMain(
+            @PathVariable("questioningId") Long questioningId,
+            @PathVariable("contactId") String contactId)  {
+
```

Comment:
maybe check first  if questioningId match an exisiting entity (case for ContactNotFound is already handled)